src/Tizen.NUI/src/public/Events/GestureOptions.cs

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using Tizen.NUI.BaseComponents;
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// This is a calss that sets the configuration options of Gestures
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public sealed class GestureOptions
+    {
+        private static readonly GestureOptions instance = new GestureOptions();
+
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        private GestureOptions()
+        {
+        }
+
+        /// <summary>
+        /// Gets the singleton of the GestureOptions object.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static GestureOptions Instance
+        {
+            get
+            {
+                return instance;
+            }
+        }
+
+        /// <summary>
+        /// Sets the prediction mode for pan gestures <br />
+        ///    * 0 - No prediction <br />
+        ///    * 1 - Prediction using average acceleration <br />
+        /// </summary>
+        /// <remarks>This is a global configuration option. Affects all gestures.</remarks>
+        /// <param name="mode">The prediction mode</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetPanGesturePredictionMode(int mode)
```

Comment:
This is hidden-API, so please consider to change the followings when you upload a patch next time.
- change `int mode` argument as enum.
- for this kind of simple API, it would be good to change to a property.